ecl/provider_connectivity/v2/tenant_connections/testing/fixtures.go

```diff
@@ -0,0 +1,639 @@
+const CreateAttachBaremetalServerRequest = `
+{
+    "tenant_connection": {
+        "name": "attach_bare_name",
+        "description": "attach_bare_desc",
+		"tags": {
+			"test_tags1": "test1"
+		},
+		"tenant_connection_request_id": "147c4ffa-481e-11ea-8088-525400060300",
+		"device_type": "ECL::Baremetal::Server",
+		"device_interface_id": "46eb7624-d462-46c2-8ac7-f988a15d3280",
+		"device_id": "0acab22f-8993-451c-8a6b-398b0244f578",
+		"attachment_opts": {
+			"segmentation_id": 10,
+			"segmentation_type": "flat",
+			"fixed_ips": [
+				{
+					"ip_address": "192.168.1.1",
+					"subnet_id": "1f424165-2202-4022-ad70-0fa6f9ec99e1"
+				}
+			],
+			"allowed_address_pairs": [
+				{
+					"ip_address": "192.168.1.2",
+					"mac_address": "11:22:33:aa:bb:cc"
+				}
+			]
+		}
+    }
+}
+`
+
+// CreateAttachBaremetalServerResponse provides the output from a Create request.
+const CreateAttachBaremetalServerResponse = `
+{
+    "tenant_connection":{
+        "id": "0d956a2e-4958-11ea-8088-525400060300",
+        "tenant_connection_request_id": "147c4ffa-481e-11ea-8088-525400060300",
+        "name": "attach_bare_name",
+        "description": "attach_bare_desc",
+        "tags": {
+			"test_tags1": "test1"
+		},
+        "tenant_id": "7e91b19b9baa423793ee74a8e1ff2be1",
+        "name_other": "",
+        "description_other": "",
+        "tags_other": {},
+        "tenant_id_other": "c7f3a68a73e845d4ba6a42fb80fce03f",
+        "network_id": "061dbaa9-a3e0-4343-b3fc-0a619db66854",
+        "device_type": "ECL::Baremetal::Server",
+        "device_id": "0acab22f-8993-451c-8a6b-398b0244f578",
+        "device_interface_id": "46eb7624-d462-46c2-8ac7-f988a15d3280",
+        "port_id": "87449d66-4e99-4cf7-9b93-9f153548ccc7",
+        "status": "creating"
+    }
+}
+`
+
+// CreateAttachVnaRequest provides the input to a Create request.
+const CreateAttachVnaRequest = `
+{
+    "tenant_connection": {
+        "name": "attach_vna_name",
+        "description": "attach_vna_desc",
+		"tags": {
+			"test_tags1": "test1"
+		},
+		"tenant_connection_request_id": "67d76b00-3804-11ea-8088-525400060300",
+		"device_type": "ECL::VirtualNetworkAppliance::VSRX",
+		"device_interface_id": "interface_2",
+		"device_id": "c291f4c4-a680-4db0-8b88-7e579f0aaa37",
+		"attachment_opts": {
+    		"fixed_ips": [
+        		{
+            		"ip_address": "192.168.1.3"
+        		}
+    		]
+		}
+    }
+}
+`
+
+// CreateAttachVnaResponse provides the output from a Create request.
+const CreateAttachVnaResponse = `
+{
+    "tenant_connection":{
+        "id": "f6331886-3804-11ea-95a8-525400060400",
+        "tenant_connection_request_id": "67d76b00-3804-11ea-8088-525400060300",
+        "name": "attach_vna_name",
+        "description": "attach_vna_desc",
+        "tags": {
+			"test_tags1": "test1"
+		},
+        "tenant_id": "7e91b19b9baa423793ee74a8e1ff2be1",
+        "name_other": "",
+        "description_other": "",
+        "tags_other": {},
```

Comment:
??
```suggestion
        "tags_other": "",
```

Reply:
tags_other is not a string, so I guess it works fine with {}.